go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	code.cloudfoundry.org/go-loggregator v7.4.0+incompatible
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	github.com/Azure/azure-event-hubs-go/v3 v3.1.2
-	github.com/Azure/azure-sdk-for-go v37.1.0+incompatible
+	github.com/Azure/azure-sdk-for-go v55.2.0+incompatible
```

Comment:
How are those related to this PR?

Reply:
This was the Lint error. That's why I added these entries.
[2021-09-09T07:13:11.517Z] Error determining list of magefiles: failed to list non-mage gofiles: exit status 1: go: github.com/prometheus/prometheus@v1.8.2-0.20210701133801-b0944590a1c9 requires
[2021-09-09T07:13:11.517Z] 	github.com/Azure/azure-sdk-for-go@v55.2.0+incompatible: missing go.sum entry; to add it:
[2021-09-09T07:13:11.517Z] 	go mod download github.com/Azure/azure-sdk-for-go

Reply:
Please check this one also 
https://beats-ci.elastic.co/job/Beats/job/beats/job/PR-27269/22/consoleText

Reply:
So it is because of the upgrade of Prometheus library, right? If that's the case then we are fine.

Reply:
Right, this change is because of Prometheus upgrade.